### ISSUE TITLE
`gp open` flag to open side-by-side (or some workaround to disable vscode enablePreview setting)

### ISSUE THREAD

**Issue by mcgrealife** [gitpod-io/gitpod#16624]
## Describe the behaviour you'd like
- I'd like to use `gp open` to open two files in side-by-side panes [vscode docs](https://code.visualstudio.com/docs/getstarted/userinterface#_side-by-side-editing)
- but currently, when gp open the second file, it replaces the first
- ideally a gp open --flag  would be great

## Describe alternatives you've considered
I tried these vscode cli commands, but they were blocked by `gitpod-code`
- `code --disable-preview` returns `Ignoring option 'disable-preview': not supported for gitpod-code.`
- `code --move-editor right` returns `Ignoring option 'move-editor': not supported for gitpod-code.`

**Comment by axonasif** [comment]
👋 @mcgrealife thanks for raising this issue!

> I tried these vscode cli commands, but they were blocked by gitpod-code
> - code --disable-preview returns Ignoring option 'disable-preview': not supported for gitpod-code.
> - code --move-editor right returns Ignoring option 'move-editor': not supported for gitpod-code.

I don't think the `code` CLI interface recognizes any such flags. You can run `code --help` to see.

**Comment by akosyakov** [comment]
gp open does not implement editor specific functionality, please use code instead

**Comment by akosyakov** [comment]
@jeanp413 for unsupported options not sure whether we could contribute it to upstream, could we open issues?

**Comment by mcgrealife** [comment]
Hi all, wow thanks for the support!

@akosyakov I have since confirmed that the Vscode CLI does not offer any `code open` commands to open a file side-by-side (I tried all the `code open` flags without success).

@axonasif you are correct, these flags do not exist `code --move-editor right` (I accidentally trusted chatgpt 😬 sorry about that)

**The only other method I've found is a vscode keyboard shortcut, "View: Move Editor into Next Group" i.e. `workbench.action.moveEditorToNextGroup`, on a mac is `ctrl command right arrow`.** But I assume triggering a macro like that might require browser permissions, not sure.

As a workaround, I added a .gif in my [readme](https://github.com/mcgrealife/edgdedb-node-ts-playground-template) recommending users to drag file side-by-side.

Thanks again for your support, what a great repository!
